docs/manual/src/tutorial/foreign_language_bindings.md

```diff
@@ -10,49 +10,48 @@ Ideally you would then run the `uniffi-bindgen` binary from the `uniffi` crate t
 is only available with [Cargo nightly](https://doc.rust-lang.org/cargo/reference/unstable.html#artifact-dependencies).
 To work around this, you need to create a binary in your project that does the same thing.
 
-
 Add the following to your `Cargo.toml`:
 
 ```toml
 [[bin]]
-name = "run-uniffi-bindgen"
-path = "run-uniffi-bindgen.rs"
+name = "uniffi-bindgen"
```

Comment:
The name isn't important, right? Maybe a comment saying something like "# whatever name makes sense for your project can be used, but that the rest of this tutorial assumes `uniffi-bindgen`"?